hack/conformance-test.sh

```diff
@@ -18,22 +18,29 @@
 # against an already-setup cluster for which there is no automated
 # setup, teardown, and other cluster/... scripts.
 #
-# The user must export these environment variables:
-# KUBE_MASTER_IP to the ip address of the master.
-# AUTH_CONFIG to the argument of the "--auth_config=" flag.
-# If certs required, set CERT_DIR.
-# 
-# Example to test against a local vagrant cluster:
-# declare -x AUTH_CONFIG="$HOME/.kubernetes_vagrant_auth"
-# declare -x KUBE_MASTER_IP=10.245.1.2
-# hack/conformance-test.sh
-if [[ -z "KUBE_MASTER_IP" ]]; then
-  echo "Must set KUBE_MASTER_IP before running conformance test."
-  exit 1
-fi
-if [[ -z "AUTH_CONFIG" ]]; then
-  echo "Must set AUTH_CONFIG before running conformance test."
-  exit 1
-fi
+# The user must:
+#  - Provide a Kubeconfig file whose current context is set to the
+#    cluster to be tested.
+#  - Specify the location of that kubeconfig with, e.g.:
+#    declare -x KUBE_CONFIG_FILE="=$HOME/.kube/config"
+#  - Specify the location of the master with, e.g.:
+#    declare -x KUBE_MASTER_IP="1.2.3.4"
+#  - Make sure only essential pods are running and there are no failed/pending pods.
+# Then run:
+#    hack/conformance-test.sh 2>&1 | tee conformance.$(date +%FT%T%z).log
+
+: ${KUBE_CONFIG_FILE:?"Must set KUBE_CONFIG_FILE before running conformance test."}
+: ${KUBE_MASTER_IP:?"Must set MASTER_IP before running conformance test."}
+
+echo "Conformance test version: 0.1"
+echo -n "Conformance test run date:"
+date
+
+# Conformance test does not have to pass all the e2es.  Just a subset for now:
+export GINKGO_TEST_ARGS="--focus=DNS|Downward|EmptyDir|Events|Kubectl|Networking|Pods|PreStop|Secrets|Services --skip=Skipped|Service\sendpoints\slatency|Services.*functioning\sexternal|Services.*Type.*LoadBalancer.*NodePort|Services.*change\sthe\stype|Service.*should.release.*|Service.*external\sload\sbalancer"
```

Comment:
`GINKGO_TEST_ARGS` is already used in `hack/jenkins/e2e.sh` to define what is passed in the `--test_args` flag to `hack/e2e.go`, which is in turn passed as arguments to `hack/ginkgo-e2e.sh`.

It's a complete roundabout mess, but using the same name here is likely to cause problems.

An alternative to creating this new env var is to just append --ginkgo.focus=... to the `hack/ginkgo-e2e.sh` line below.